pkg/services/app/add_test.go

```diff
@@ -329,7 +329,7 @@ var _ = Describe("New directory structure", func() {
 		Expect(manifestsByPath[filepath.Join(git.WegoRoot, git.WegoAppDir, addParams.Name, "kustomization.yaml")]).ToNot(BeNil())
 		cname, err := kubeClient.GetClusterName(context.Background())
 		Expect(err).To(BeNil())
-		clusterKustomizeFile := manifestsByPath[filepath.Join(git.WegoRoot, git.WegoClusterDir, cname, "/user/kustomization.yaml")]
+		clusterKustomizeFile := manifestsByPath[filepath.Join(git.WegoRoot, git.WegoClusterDir, cname, git.WegoClusterUserWorloadDir, "kustomization.yaml")]
```

Comment:
s/WegoClusterUserWorloadDir/WegoClusterUserWorkloadDir

Reply:
👍 thanks for catching this.  I saw it at one point but then forgot to fix it.